src/test/java/pages/SearchPage.java

```diff
@@ -90,4 +95,18 @@ public void iconEpmtyResults(){
         $(By.cssSelector("div.jira-adbox.jira-adbox-medium.no-results.no-results-message")).shouldBe(Condition.visible);
     }
 
+    public void searchResultsContains(String request){
+        $(".focused").shouldBe(Condition.visible).shouldHave(attribute("data-key")).shouldHave(text(request));
+    }
+
+    public void selectProject(String request){
+        $(By.cssSelector(".criteria-selector.aui-button.aui-button-subtle.drop-arrow")).click();
+        $(By.id("searcher-pid-input")).setValue(request);
+        $(By.cssSelector("label[title='" +request+"']")).click(); }
+
+    public void searchResultsTypeContains(String request){
+        $(".list-content").shouldBe(Condition.visible).$("img").shouldHave(attribute("alt", request)); }
+
+    public List<SelenideElement> issueListContainType(){
+        return $(".list-content").shouldBe(Condition.visible).$$("img");}
```

Comment:
Not easy to read. Can you divide it to different rows or use separate variable?

Reply:
I will try to change